trie/verkle.go

```diff
@@ -143,9 +143,8 @@ func (t *VerkleTrie) GetAccount(addr common.Address) (*types.StateAccount, error
 		}
 	}
 	var balance [32]byte
-	copy(balance[16:], values[utils.BasicDataLeafKey][utils.BasicDataBalanceOffset:])
-	for i := 0; i < 8; i++ {
-		balance[31-i], balance[utils.BasicDataBalanceOffset+i] = balance[utils.BasicDataBalanceOffset+i], balance[31-i]
+	for i, b := range values[utils.BasicDataLeafKey][utils.BasicDataBalanceOffset:] {
+		balance[32-i-1] = b
```

Comment:
this means the balance is still little endian, but at least it's consistent. We can fix it here, or in another PR, it doesn't matter.